editor/plugins/tile_map_editor_plugin.cpp

```diff
@@ -2055,7 +2047,12 @@ TileMapEditor::TileMapEditor(EditorNode *p_editor) {
 
 	// Tile position.
 	tile_info = memnew(Label);
-	toolbar_right->add_child(tile_info);
+	tile_info->set_modulate(Color(1, 1, 1, 0.8));
+	tile_info->set_mouse_filter(MOUSE_FILTER_IGNORE);
```

Comment:
Not a critical issue, but I noticed that despite this flag, I still can't place tiles behind the label. This also occurs with the [`warning_child_of_container`](https://github.com/Calinou/godot/blob/13f98f80b62f736f36de8b75722901d8dd8311a7/editor/plugins/canvas_item_editor_plugin.cpp#L5354-L5359) info label.

Reply:
Dumb question, but have you tried `MOUSE_FILTER_PASS`?

Reply:
@homer666 I have, but it still blocks mouse input behind the label.

Reply:
When using `add_control_to_info_overlay()`, it adds it to a `VBoxContainer`, could that be blocking the mouse?
https://github.com/godotengine/godot/blob/d422eff03604ff5ea10692e71c92f0d79a6d9355/editor/plugins/canvas_item_editor_plugin.cpp#L5185-L5191

Reply:
@YeldhamDev Don't Containers have their mouse mode set to Pass (or Ignore) by default? I remember seeing this when I used Zylann's editor debuggeer plugin.